content/reviewers_response.md

```diff
@@ -0,0 +1,665 @@
+
+**We recognize that this section is a bit dense and have updated our manuscript to unpack this explanation.**
+
+```diff
++ Training models to identify which journal publishes similar articles is challenging as not all journals are the same. 
++ Some journals have a publication rate of at most hundreds of papers per year, while others publish at a rate of at least ten thousand papers per year.
++ Furthermore, some journals focus on publishing articles within a concentrated topic area, while others cover many dispersive topics.
++ Therefore, we designed two approaches to account for these characteristics.
++ Our first approach focuses on articles that account for a journal's variation of publication topics.
++ This approach allows for topically similar papers to be retrieved independently of their respective journal.
++ Our second approach is centered on journals to account for varying publication rates.
++ This approach allows more selective or less popular journals to have equal representation to their high publishing counterparts.
+```
+
+> I'm also curious how often these two classifiers agree--are the top matching papers typically found in the top matching journals? In cases where the two classifiers tend to disagree, are there any common characteristics of the preprints the application is trying to classify?
+
+**We evaluated our classifier agreement by calculating the overlap coefficient, which is designed to measure the overlap between two sets.
+We randomly sampled 1700 out of 20232 known preprint-published pairs from our test dataset.
+We generated ten recommendations from our centroid classifier and ten unique journal recommendations from our paper-paper model for every sampled pair. 
+This model achieved an average overlap coefficient of 0.21.
+Along with this calculation, we generated two random baselines for both our models.
+Our first baseline was designed for our journal centroid model.
+We subsetted our preprint-published pairs to contain a distinct listing of each unique journal, then randomly sampled ten journals without replacement for each preprint-published pair.
+We compared this random listing against our original journal centroid recommendation list and achieved an average score of 0.0184.
+Our other baseline was designed for our paper-paper classifier.
+We randomly sampled without filtering ten unique journals for each preprint-published pair and compared this sample to the original paper-paper recommendation list. 
+This baseline resulted in an average overlap coefficient of 0.009.
+Overall, our approach has better recommendation agreement compared to our randomly sampled baseline.**
```

Comment:
```suggestion
Our takeaway from this analysis is that both approaches agree much more than they would due to random overlap, but the overlap coefficient remains modest.
Because of the relatively large number of discrepancies between the resulting sets, we were not able to identify a practical way to answer the characteristics of preprints that led to differences vs common predictions.**
```